docs/src/main/asciidoc/undertow-reference.adoc

```diff
@@ -29,3 +29,39 @@ jar files as defined by the Servlet spec.
 You can make use of the Undertow predicate language using an `undertow-handlers.conf` file. This file should be placed
 in the `META-INF` directory of your application jar. This file contains handlers defined using the
 link:http://undertow.io/undertow-docs/undertow-docs-2.0.0/index.html#predicates-attributes-and-handlers[Undertow predicate language].
+
+== CORS filter
+
+Quarkus comes with a CORS filter which can be enabled in the Quarkus configuration file, `application.properties`:
+
+[source, properties]
+----
+quarkus.http.cors=true
+----
+
+[NOTE]
+====
+The `application.properties` file must be located in the `src/main/resources` source folder. 
```

Comment:
why not use directly `src/main/resources/application.properties` above?

That would avoid the note.